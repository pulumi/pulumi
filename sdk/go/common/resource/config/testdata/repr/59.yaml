my:servers:
- host: example
  name: foo
