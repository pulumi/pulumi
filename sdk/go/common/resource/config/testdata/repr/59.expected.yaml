decrypt:
  my:servers: '[{"host":"example","name":"foo"}]'
paths:
  my:servers:
    value:
    - host: example
      name: foo
    string: '[{"host":"example","name":"foo"}]'
    redacted: '[{"host":"example","name":"foo"}]'
    object: HRAADltdaW50ZXJmYWNlIHt9/4MCAQL/hAABEAAAW/+EKAABF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAv/4IsAAIEaG9zdAZzdHJpbmcMCQAHZXhhbXBsZQRuYW1lBnN0cmluZwwFAANmb28=
    secure: false
    isObject: true
  my:servers[0]:
    value:
      host: example
      name: foo
    string: '{"host":"example","name":"foo"}'
    redacted: '{"host":"example","name":"foo"}'
    object: KBAAF21hcFtzdHJpbmddaW50ZXJmYWNlIHt9/4EEAQL/ggABDAEQAAAv/4IsAAIEbmFtZQZzdHJpbmcMBQADZm9vBGhvc3QGc3RyaW5nDAkAB2V4YW1wbGU=
    secure: false
    isObject: true
  my:servers[0].host:
    value: example
    string: example
    redacted: example
    object: FBAABnN0cmluZwwJAAdleGFtcGxl
    secure: false
    isObject: false
  my:servers[0].name:
    value: foo
    string: foo
    redacted: foo
    object: EBAABnN0cmluZwwFAANmb28=
    secure: false
    isObject: false
